README.md

```diff
@@ -254,12 +254,33 @@ cd genny
 evergreen set-module -m dsi -i <ID> # Use the build ID from the previous step.
 ```
 
-In the browser window, select the workloads you wish to run. Good
-examples are Linux Standalone / `big_update` and Linux Standalone /
-`insert_remove`.
+In the browser window, select either `genny_patch_tasks` or `genny_auto_tasks`.
+`genny_patch_tasks` will run any workloads that you have added or modified locally
+(based on your git history). This is useful if you want to test only the workload(s)
+you've been working on. 
+
+`genny_auto_tasks` automatically runs workloads based on the evergreen environment
+(variables from `bootstrap.yml` and `runtime.yml`) and an optional AutoRun section
```

Comment:
Suggest adding "in DSI" at the end of the parenthetical.

Reply:
Done.